### PR TITLE
Update surelog

### PR DESCRIPTION
**Pull request by RRozak**
It changes include statements required by https://github.com/chipsalliance/UHDM/pull/488

### CODE REVIEW

**Comment by RRozak** [comment]
Some of the errors can be reproduced in https://github.com/chipsalliance/UHDM-integration-tests/pull/461

**Comment by RRozak** [comment]
Blocked by chipsalliance/Surelog#1866

File: build_binaries.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -ex
 cd Surelog && make PREFIX=$PWD/../image/ release install -j $(nproc) && cd ..
-autoconf && ./configure --prefix=$PWD/image && make -j $(nproc) && make install
+autoconf && ./configure --prefix=$PWD/image SURELOG_INSTALL_DIR=$PWD/image && make -j $(nproc) && make install
```
Comment (rkapuscik) [inline build_binaries.sh]: How about we just set `SURELOG_INSTALL_DIR` at the beginning and then feed it to Surelog's `PREFIX`?

**Review by rkapuscik — APPROVED** [review]

File: build_binaries.sh (hunk 1 above)
Comment (RRozak) [reply build_binaries.sh]: Done

**Review by rkapuscik — APPROVED** [review]

**Pull request merged**